trailblazer/services/job_service/job_service.py

```diff
@@ -64,8 +71,24 @@ def get_analysis_status(self, analysis_id: int) -> TrailblazerStatus:
         if not analysis.jobs:
             raise NoJobsError(f"No jobs found for analysis {analysis_id}")
 
-        return get_status(analysis.jobs)
+        status: TrailblazerStatus = get_status(analysis.jobs)
+        if (
+            analysis.workflow_manager == WorkflowManager.TOWER
+            and status == TrailblazerStatus.COMPLETED
+        ):
+            status = TrailblazerStatus.QC
+        return status
```

Comment:
For the future, we could use `workflow.onComplete` and `workflow.onError` statements within tower itself. This allow to trigger something or run some code after the pipeline is succesful or if it exists with errors. We could transfer the logic there if that's preferred and set the trailblazer status on pipeline completion. Anyways, this is not part of this PR and I'm not sure we want that either (would need to be thought through)

Reply:
That is nice! I think that could make sense, unless we want to assume that all workflows run via Tower in the future always will have a separate QC step.

Alternatively, we could track the QC job directly on the analysis in trailblazer somehow.